### ISSUE TITLE
[bug] bud-error overlay obscures part of the window when inactive

### ISSUE THREAD

**Issue by mensch** [roots/bud#1597]
### Agreement

- [X] This is not a duplicate of an existing issue
- [X] I have read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

### Describe the issue

When running `yarn dev` bud adds an overlay element to the shadow dom. The positioning of this overlay isn't much of an issue on smaller viewports, but with larger ones part of the overlay element obscures part of the viewport of the browser. Elements underneath the overlay can't be clicked and the "Select an element" option of the developer tools doesn't work properly anymore as well.

The inactive overlay has the following styles:

```css
.overlay {
    width: 100vw;
    backdrop-filter: blur(10px);
    display: flex;
    height: 100vh;
    border-top: 2px solid transparent;
    overflow-x: hidden;
    overflow-y: scroll;
    position: absolute;
    top: -1000px;
    left: 0;
    right: 0;
    bottom: 0;
    opacity: 0;
    transition: opacity 0.2s ease-in-out, border 0.4s ease-in-out;
    justify-content: center;
}
```
The `top` rule should probably be a relative value instead of a fixed one.

### Expected Behavior

The overlay should be completely hidden on any screen dimenions when inactive.

### Actual Behavior

The overlay obscures part of the viewport.

### Steps To Reproduce

1. Install bud 6.3.3
2. Run yarn dev
3. Load development URL in browser

### version

6.3.3

### What package manager are you using?

yarn

### version

3.2.1

**Comment by mensch** [comment]
Also reported [here by another user](https://discourse.roots.io/t/empty-bud-error-overlay-preventing-pointer-events/23592) on the Sage Discourse.

**Comment by kellymears** [comment]
Thanks. I think the solution is to actually inject the element only when it's needed and then remove it from the DOM when errors are resolved. I'll get a patch going for that.

**Comment by kellymears** [comment]
this should be resolved in 6.3.4

**Comment by mensch** [comment]
I can confirm this is resolved by upgrading to 6.3.4.
